src/org/robockets/steamworks/commands/Shoot.java

```diff
@@ -0,0 +1,37 @@
+package org.robockets.steamworks.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.robockets.commons.YAxisRelativeDirection;
+import org.robockets.steamworks.Robot;
+
+/**
+ * @author Jake Backer
+ */
+public class Shoot extends Command {
+
+    public Shoot() {
+
+    }
+
+    protected void initialize() {
+
+    }
+
+    protected void execute() {
+        Robot.shooter.spinUp();
```

Comment:
This shouldn't be in initialize because it will be running the motors which require to constantly be running therefore putting it in initialize would not work